src/core/js/models/itemsComponentModel.js

```diff
@@ -0,0 +1,76 @@
+define([
+    'core/js/models/componentModel',
+    'core/js/models/itemModel'
+], function(ComponentModel, ItemModel) {
+
+    var ItemsComponentModel = ComponentModel.extend({
+
+        toJSON: function() {
+            var json = _.clone(this.attributes);
+
+            json._items = this.get('_items').toJSON();
+
+            return json;
+        },
+
+        init: function() {
+            this.setUpItems();
+
+            this.listenTo(this.get('_items'), {
+                'change:_isVisited': this.checkCompletionStatus
+            });
+        },
+
+        setUpItems: function() {
+            var items = this.get('_items');
+            var collection = new Backbone.Collection(null, { model: ItemModel });
+
+            items.forEach(function(item, index) {
```

Comment:
You could actually call `.map()` here and call the Collection `add()` with an array of `items` after.